tsdb/engine/tsm1/float_test.go

```diff
@@ -0,0 +1,165 @@
+package tsm1_test
+
+import (
+	"testing"
+
+	"github.com/influxdb/influxdb/tsdb/engine/tsm1"
+)
+
+func TestFloatEncoder_Simple(t *testing.T) {
```

Comment:
Why no test data of actual float data? I know it's all converted to floats, but it would be nice to sure everything works OK when the values pushed in are actually floats. This would make it crystal clear to the next guy reading the code that it really does support floats.